Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -110,7 +114,8 @@ CGFontRef CGFontCreateWithDataProvider(CGDataProviderRef cgDataProvider) {
     CFAutorelease(ret);
     struct __CGFont* mutableRet = const_cast<struct __CGFont*>(ret);
 
-    RETURN_NULL_IF_FAILED(_DWriteCreateFontFaceWithDataProvider(cgDataProvider, &mutableRet->_dwriteFontFace));
+    mutableRet->_data.reset(CGDataProviderCopyData(cgDataProvider));
+    RETURN_NULL_IF_FAILED(_DWriteCreateFontFaceWithData(mutableRet->_data.get(), &mutableRet->_dwriteFontFace));
```

Comment:
Seems like this should be FAIL_FAST? Ignoring a dwrite error seems to go against the pattern of failing at the public api level that we've been following.